backend/.eslintrc.js

```diff
@@ -1,27 +1,33 @@
 module.exports = {
-    parser: "@typescript-eslint/parser", // Specifies the ESLint parser
-    extends: [
-      "plugin:@typescript-eslint/recommended", // Uses the recommended rules from @typescript-eslint/eslint-plugin
-      "plugin:prettier/recommended",
-    ],
-    parserOptions: {
-      ecmaFeatures: {
-        jsx: true // Allows for the parsing of JSX
-      }
+  parser: "@typescript-eslint/parser", // Specifies the ESLint parser
```

Comment:
I am aware that I introduced all those comments by copy pasting a default config file, but they are really not useful at all in this file.

Reply:
removed